parser/parser.y

```diff
@@ -2137,6 +2139,22 @@ FunctionCallNonKeyword:
 	{
 		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr($1.(string))}
 	}
+|	"CURTIME" FuncDatetimePrec
```

Comment:
As "CURTIME" is synonym of "CURRENT_TIME", we can implement one of them, and parse the other one to the same type of token in `scanner.l`, like this PR:
https://github.com/pingcap/tidb/pull/728